### ISSUE TITLE
Answers file templating fails when copy message is provided

### ISSUE THREAD

**Issue by MigQ2** [copier-org/copier#1825]
### Describe the problem

The [answers file templating](https://github.com/copier-org/copier/pull/1027) fails when using a `_message_before_copy` and `_message_after_copy`

If the messages are removed the answers file gets correctly templated

I don't know how the copier internals work well enough to figure out why this is happening. Any ideas?

### Template

I created a dummy template in https://github.com/MigQ2/dummy-copier-template, with the `master` branch working fine but the [fix/broken-template](https://github.com/MigQ2/dummy-copier-template/tree/fix/broken-template) branch doesn't work.

The only difference between both branches is the  `_message_before_copy` and `_message_after_copy`, you can check that [here](https://github.com/MigQ2/dummy-copier-template/pull/1/files)

### To Reproduce

I am using `copier==9.4.1`

You can run `copier copy -d dummy_answer="myanswer" https://github.com/MigQ2/dummy-copier-template.git .` and see that the copier answers file gets correctly templated:

![image](https://github.com/user-attachments/assets/a722e0b4-857e-4cdc-b692-f4110afad6e8)

Then run `copier copy --vcs-ref fix/broken-template  -d dummy_answer="myanswer" https://github.com/MigQ2/dummy-copier-template.git .` and you will see the copier answers file doesn't resolve the template:

![image](https://github.com/user-attachments/assets/34193ecc-f858-499c-bd58-d1bc6e727527)

### Expected behavior

Answer file should be correctly templated when using a message 

### Operating system

Linux

### Operating system distribution and version

Ubuntu 22.04.3 LTS (running on WSL)

### Copier version

9.4.1

### Python version

3.10

### Installation method

pip+pypi

**Comment by sisp** [comment]
Thanks for reporting this bug, @MigQ2! :pray: I've been able to reproduce the bug (also on the current tip of the `master` branch) and submitted a PR that fixes it: #1840

**Comment by cbrown1234** [comment]
I had been running into this issue, and couldn't work out what was causing it for ages until I finally realised it was `_message_before_copy`. As I was looking into it to try to fix it, I came across your test, thanks for fixing @sisp!

Could I ask what's the release cadence? I'm interested when this (and another PR I've submitted https://github.com/copier-org/copier/pull/1892) could be available for users of our templates
